README.md

```diff
@@ -132,22 +129,22 @@ After copying the 'vexim' directory, you should find the 'variables.php.example'
 * $sqlpass – to the vexim database user's password which you chose while editing 'mysql.sql' in the "Databases and authentication" step.
 * $uid, $gid and $mailroot to the values you have from the "System user" step.
 * $cryptscheme is set to "sha512", a more specific configuration or other crypt-schemes can be used.
+* The URL to mailman in `$mailmanroot`
```

Comment:
why not write this the same way as other lines?
And if you want to add backticks, please add them to all of these variable names, not just this one.